aviary/interface/methods_for_level2.py

```diff
@@ -274,24 +115,40 @@ def load_inputs(
         """
         # compatibility with being passed int for verbosity
         verbosity = Verbosity(verbosity)
+
         ## LOAD INPUT FILE ###
         # Create AviaryValues object from file (or process existing AviaryValues object
         # with default values from metadata) and generate initial guesses
-        aviary_inputs, initialization_guesses = create_vehicle(
+        aviary_inputs, self.initialization_guesses = create_vehicle(
             aviary_inputs, meta_data=meta_data, verbosity=verbosity)
+        self.aviary_inputs = aviary_inputs
 
         # pull which methods will be used for subsystems and mission
         self.mission_method = mission_method = aviary_inputs.get_val(
             Settings.EQUATIONS_OF_MOTION)
         self.mass_method = mass_method = aviary_inputs.get_val(Settings.MASS_METHOD)
 
+        # Determine which problem builder to use based on mission_method
+        if mission_method is HEIGHT_ENERGY:
+            self.builder = ProblemBuilderHeightEnergy()
+        elif mission_method is TWO_DEGREES_OF_FREEDOM:
+            self.builder = ProblemBuilder2DOF()
+        elif mission_method is SOLVED_2DOF:
+            self.builder = ProblemBuilderSolved2DOF()
+        elif mission_method is CUSTOM:
+            if problem_builder:
+                self.builder = problem_builder()
+                # TODO: make draft / example custom builder
+            else:
+                raise ValueError(
+                    f'When using "settings:equations_of_motion,custom", a problem_builder must be specified in load_inputs().')
+        else:
+            raise ValueError(
+                f'settings:equations_of_motion must be one of: height_energy, 2DOF, or custom')
```

Comment:
SOLVED_2DOF should be added here per line 136

Reply:
Added